### PR TITLE
first implemetation

### PR DESCRIPTION
**Pull request by mansinayakpb**
1. Create the models (user, task, comment)
2.  register the models in admin file
3. create form(signupform, loginform, task, comment)
4. create urls
5. create the views(homeview, signupview, loginview, logoutview)
6. add the templates file

### CODE REVIEW

File: app/forms.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            "detail",
+            "due_date",
+            "status",
+            "priority",
+            "assigned_by",
+            "assigned_to",
+        ]
+
+
+class CommentForm(forms.ModelForm):
```
Comment (tanishgupta905) [inline app/forms.py:38]: what is the use case of this model

**Review by tanishgupta905 — APPROVED** [review]

**Comment by tanishgupta905** [comment]
fix flake error in your code

File: app/forms.py (hunk 1 above)
Comment (mansinayakpb) [reply app/forms.py:38]: Sir, CommentForm use the fields from models, this model is going to create, update the comments on a specific task
so, here i have used task as a foreign key relationship so, if user wants to comment or discuss something on regarding the task i.e., why i have created this form.

Comment (tanishgupta905) [reply app/forms.py:38]: makes sense!

File: app/models.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
+from django.db import models
+from django.utils import timezone
+from django.contrib.auth.models import AbstractUser
+
+# Create your models here.
+
+
+class TimesStampedModel(models.Model):
```
Comment (rajatShr) [inline app/models.py:8]: No need to create the TimeStampedModel model, just import this
from django_extensions.db.models import TimeStampedModel

**Review by rajatShr — APPROVED** [review]
Please add gitignore file and add __pycache__

File: app/models.py (hunk 2 above)
Comment (mansinayakpb) [reply app/models.py:8]: Ohk Sir, should i need to remove TimeStampedModel.

**Review by tanishgupta905 — APPROVED** [review]

**Pull request merged**